### PR TITLE
Adding GuLoader template

### PR DESCRIPTION
**Pull request by gio-pecora**
Adding GuLoader template with indicators up to 2021-Dec-02

## Description of the change
New Template

## Type of change
New feature (non-breaking change that adds functionality)

### CODE REVIEW

File: Emerging_Threats/guloader.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    op: ends with
+    path: event/HASH
+    value: 4366a0e113d168f2809a4a1983c2198ec874b89af0bdfe465e753d409c85c51c
+  respond:
+    - action: report
+      name: 'loki-guloader-2021-12-02 (https://attack.mitre.org/software/S0561/)'
+'GuLoader #006':
+  namespace: general
+  detect:
+    event: ONGOING_IDENTITY
```
Comment (maximelb) [inline Emerging_Threats/guloader.yml]: Those should be targeting `CODE_IDENTITY`. The `ONGOING_IDENTITY` is not reported to the cloud by default.
```suggestion
    event: CODE_IDENTITY
```

File: Emerging_Threats/guloader.yml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    op: ends with
```
Comment (maximelb) [inline Emerging_Threats/guloader.yml]: Hashes are always a specific length, so we should use `is` for hashes.
```suggestion
    op: is
```

File: Emerging_Threats/guloader.yml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    respond:
+      - action: report
+        name: guloader-2021-12-02
+  'GuLoader #002':
+    namespace: general
+    detect:
+      event: DNS_REQUEST
+      op: is
+      path: event/DOMAIN_NAME
+      value: nowautomation.com
```
Comment (maximelb) [inline Emerging_Threats/guloader.yml:22]: Is it just this domain that's bad, or also possibly subdomains?
If subdomains should be included you could to:
```yaml
event: DNS_REQUEST
op: is
path: event/DOMAIN_NAME
value: nowautomation.com
sub domain: '-2:'
```

as documented here: https://doc.limacharlie.io/docs/documentation/ZG9jOjE5OTA0MjQ0-reference-operators#sub-domain

**Review by maximelb — CHANGES_REQUESTED** [review]
Also generally, it might be nice to group the hashes relating to the same dropper together in one rule that uses OR, then the same for DNS relating to the same one.

Makes it nicer to have one logical rule that reports a general threat based on any number of indicators.

File: Emerging_Threats/guloader.yml (hunk 3 above)
Comment (gio-pecora) [reply Emerging_Threats/guloader.yml:22]: https://refractionpoint.slack.com/archives/C02DNHSF8JF/p1638549218167000?thread_ts=1638478408.157200&cid=C02DNHSF8JF

File: Emerging_Threats/guloader.yml (hunk 1 above)
Comment (gio-pecora) [reply Emerging_Threats/guloader.yml]: I agree,
I was using the as reference the previous template pushed/reviewed on the repo:
https://github.com/refractionPOINT/rules/blob/617b4272c7a07cfb20e3dc1c7aa8a0b1fcdc3c71/Emerging_Threats/blackmatter_ransomware.yml#L39

File: Emerging_Threats/guloader.yml (hunk 2 above)
Comment (gio-pecora) [reply Emerging_Threats/guloader.yml]: I agree,
I was using the as reference the previous template pushed/reviewed on the repo:
https://github.com/refractionPOINT/rules/blob/617b4272c7a07cfb20e3dc1c7aa8a0b1fcdc3c71/Emerging_Threats/blackmatter_ransomware.yml#L39

**Review by maximelb — APPROVED** [review]

**Pull request merged**